### ISSUE TITLE
Move GraphiQL menu under Search

### ISSUE THREAD

**Issue by ns8482e** [OrchardCMS/OrchardCore#12710]
### Is your feature request related to a problem? Please describe.
Move GraphiQL menu under parent menu Search. It's really a Search and not Configuration.

**Comment by Skrypt** [comment]
Well technically it is a Query not a Search.

**Comment by hishamco** [comment]
Agree with @Skrypt, it's a query, we could add a new menu item if it's not there named `Queries`, then we could add `GraphQL`, `SQL` .. ect

**Comment by ns8482e** [comment]
> Well technically it is a Query not a Search.

Currently Queries are under Search

**Comment by hishamco** [comment]
I forgot that, seems that's why Niraj suggested that  :)

**Comment by Skrypt** [comment]
Queries are either Search or SQL related this is why they are regrouped together. Wait before anything. Let's discuss about it.

**Comment by hishamco** [comment]
Search requires query on some data store but I prefer to group search methodologies into a menu item called `Search` this should include Lucene, Elasticsearch, Azure Search .. etc. On other hand we need to have `Queries` which may include SQL, GraphQL .. etc

**Comment by ns8482e** [comment]
Queries on the other hand also saved into db to execute later

**Comment by MikeAlhayek** [comment]
I agree with @ns8482e. It should not be under configurations. under "Search" is a good place for it since it's just another way to search contents. "Search" from user perspective is not only Lucene or Elastic search. I also think anything configuring the search engines should be moved under `Configuration >> Search`. Either way, placing GraphQL menu under "Search" make scene to me.

**Comment by MikeAlhayek** [comment]
@Piedone @sebastienros thoughts on this?

**Comment by gvkries** [comment]
I think the new "Tools" menu as suggested in #17112 makes even more sense than "search".

**Comment by Piedone** [comment]
Agree.
